packages/admin/cms-admin/src/dam/DataGrid/DamContextMenu.tsx

```diff
@@ -6,6 +6,7 @@ import { saveAs } from "file-saver";
 import * as React from "react";
 import { FormattedMessage } from "react-intl";
 
+import { useCmsBlockContext } from "../../blocks/useCmsBlockContext";
```

Comment:
Out of scope: @thomasdax98 it's kinda strange that the DAM uses the CMS block context internally